CMakeLists.txt

```
@@ -1,5 +1,5 @@
 # Minimum version required
-cmake_minimum_required (VERSION 3.2)
+cmake_minimum_required (VERSION 3.2...3.5)
```

```suggestion
cmake_minimum_required (VERSION 3.5)
```

I'd rather just bump the minimum version. I can't see a reason it will be problematic to bump it only a little bit.